irr/src/CIrrDeviceSDL.cpp

```diff
@@ -220,6 +230,51 @@ int CIrrDeviceSDL::findCharToPassToIrrlicht(uint32_t sdlKey, EKEY_CODE irrlichtK
 	}
 }
 
+// Irrlicht has some EKEY_CODE entries that only appear to make sense in a SDL-scancode-like context.
+// These keycodes are passed by (in particular) the X11 IrrlichtDevice.
```

Comment:
this is a bit confusing at first, because it's not like x11 and sdl can be active at the same time.
is this for the sake of sharing configs between them?

Reply:
I dropped the the conversion code from OEM keys and scancodes because we now ask users to rebind anyway. Resolving.